wolfssl/ssl.h

```diff
@@ -1389,7 +1408,7 @@ WOLFSSL_API const char* wolfSSL_RAND_file_name(char*, unsigned long);
 WOLFSSL_API int         wolfSSL_RAND_write_file(const char*);
 WOLFSSL_API int         wolfSSL_RAND_load_file(const char*, long);
 WOLFSSL_API int         wolfSSL_RAND_egd(const char*);
-WOLFSSL_API int         wolfSSL_RAND_seed(const void*, int);
+WOLFSSL_API void        wolfSSL_RAND_seed(const void*, int);
```

Comment:
I think we should revert this change, even though it does not match openssl compatibility because the ABI tracker will flag an issue here. Its fine for an int to return where caller thinks its void, but not for existing customers who might expect a return code here.